server/test/api/costCenter.js

```diff
@@ -0,0 +1,105 @@
+/*global describe, it, beforeEach, process*/
+
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = chai.expect;
+chai.use(chaiHttp);
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
+
+var url = 'https://localhost:8080';
+var user = { username : 'superuser', password : 'superuser', project: 1};
+
+describe('The /cost_center API endpoint', function () {
+ var agent = chai.request.agent(helpers.baseUrl);
+ var newCostCenter = {
+    project_id : 1,
+    //id : 200,
+    text : 'tested cost',
+    note : 'test inserted',
+    is_principal : 1
+  };
+
+  var DELETABLE_COST_CENTER_ID = 2;
+
+  var FETCHABLE_COST_CENTER_ID = 1;
+
+   // throw errors
+  beforeEach(helpers.login(agent));
+
+    it(' A GET /cost_centers?list=full returns a list of cost centers', function () {
```

Comment:
This text should be updated to say "full=1" instead of "list=full".  A poor description for a test will not help us debug!